acquire/collector.py

```diff
@@ -520,20 +508,20 @@ def collect_path(
 
         except (FileNotFoundError, NotADirectoryError, NotASymlinkError, SymlinkRecursionError, ValueError):
             self.report.add_path_missing(module_name, error_path)
-            log.error("- Path %s is not found (while collecting %s)", error_path, path)
+            log.error("- Path %s is not found (while collecting %s)", error_path, path)  # noqa: TRY400
         except OSError as error:
             if error.errno == errno.ENOENT:
                 self.report.add_path_missing(module_name, error_path)
-                log.error("- Path %s is not found (while collecting %s)", error_path, path)
+                log.error("- Path %s is not found (while collecting %s)", error_path, path)  # noqa: TRY400
             elif error.errno == errno.EACCES:
                 self.report.add_path_failed(module_name, error_path)
-                log.error("- Permission denied while accessing path %s (while collecting %s)", error_path, path)
+                log.error("- Permission denied while accessing path %s (while collecting %s)", error_path, path)  # noqa: TRY400
             else:
                 self.report.add_path_failed(module_name, error_path)
-                log.error("- OSError while collecting path %s (while collecting %s)", error_path, path)
+                log.error("- OSError while collecting path %s (while collecting %s)", error_path, path)  # noqa: TRY400
         except Exception:
             self.report.add_path_failed(module_name, error_path)
-            log.error("- Failed to collect path %s (while collecting %s)", error_path, path, exc_info=True)
+            log.exception("- Failed to collect path %s (while collecting %s)", error_path, path)
```

Comment:
This change makes the tests fail. Reverting it to `log.error` again lets it pass again, not sure if that is preferred.

Reply:
It is not.

Reply:
But I caved in.